content/docs/tasks/security/authn-policy.md

```diff
@@ -251,14 +251,14 @@ $ kubectl exec $(kubectl get pod -l app=sleep -n legacy -o jsonpath={.items..met
 
 ## Enable mTLS incrementally
 
-Sometimes it's difficult to enable mTLS for a serivce without breaking any clients. For example, a service can have clients without Envoy sidecar. Those clients
+Sometimes it's difficult to enable mTLS for a service without breaking any clients. For example, a service can have clients without Envoy sidecar. Those clients
```

Comment:
We should focus the solution for migration only, i.e servers and clients are in different mode during transitions time.